test/utils_test.py

```diff
@@ -29,6 +34,13 @@ def test_object_name():
         check_object_name("foo/bar", "Volume", warn=True)
 
 
+def test_environment_name():
+    assert is_valid_environment_name("AB13/root:env1")
+    assert not is_valid_environment_name("--help")
+    assert not is_valid_environment_name(":env")
+    assert not is_valid_environment_name("/env")
```

Comment:
```suggestion
    assert not is_valid_environment_name("/env")
    assert not is_valid_environment_name("")
```

Reply:
Check also exercise the "not too long" rule with something like `"a" * 65`